code/datums/components/tameable.dm

```diff
@@ -44,8 +44,11 @@
 
 	var/atom/atom_parent = source
 	atom_parent.balloon_alert(attacker, "fed")
+	var/modified_tame_chance = current_tame_chance
+	if(HAS_TRAIT(attacker, TRAIT_SETTLER))
+		modified_tame_chance += 50
 	if(unique || !already_friends(attacker))
-		if(prob(current_tame_chance)) //note: lack of feedback message is deliberate, keep them guessing!
+		if(prob(modified_tame_chance)) //note: lack of feedback message is deliberate, keep them guessing!
```

Comment:
(suggestion) maybe if we're a settler we can intuitively know if animal is getting tamed? from a technical sense you'd want to cache the `HAS_TRAIT()` result and keep applying it through the proc (or rewrite it entirely) but it sound nifty. totally optional though

Reply:
wasn't hard to implement